idea/src/org/jetbrains/kotlin/idea/intentions/ConvertFunctionToPropertyIntention.kt

```diff
@@ -88,8 +85,10 @@ class ConvertFunctionToPropertyIntention : SelfTargetingIntention<KtNamedFunctio
             function.setName(newName)
 
             val property = originalFunction.replace(psiFactory.createProperty(function.text)) as KtProperty
-            if (needsExplicitType) {
-                elementsToShorten.add(property.typeReference!!)
+
+            if (!hasExplicitType) {
+                val linebreak = property.getter?.node?.treePrev
```

Comment:
I agree this isn't pretty. Currently, the formatter allows both, getters on the same line and on the next line. Forcing one of them could be to strict.

Maybe the `KtPsiFactory` could get an option to create a property with the getter on the same line. What do you think @yole?